CondTools/Geometry/test/writehelpers/splitExtended2021Database.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```

Comment:
the conddb_import command will be soon decommissioned. please replace with conddb as follows:
conddb --db [input file] [input_tag] [destination_tag] --destdb [dest_file] 
example:
conddb --db myfile.db XMLFILE_Geometry_TagXX_Extended2021_mc XMLFILE_Geometry_TagXX_Extended2021_mc --destdb GeometryFileExtended2021.db

Reply:
@ggovi - this should be done for all the scripts, not just in this PR. IMHO, it should be done at a later stage.

Reply:
@ianna 
true it should be done for all the existing scripts, but a new PR should be already compliant - we should not add something that we already know needs to be modified...